cypress/support/pages/admin/pageAdministration.js

```diff
@@ -263,25 +263,25 @@ class Management {
 		cy.wait("@search_api").its("response.statusCode").should("eq", 200);
 	}
 
-	clickEditStudentButton(email) {
+	clickEditUserButton(role, email) {
 		cy.contains("td", email)
 			.siblings()
 			.find("a")
-			.should("have.attr", "data-testid", "edit_student_button")
+			.should("have.attr", "data-testid", "edit_" + role  + "_button")
```

Comment:
you can assign this `"edit_" + role  + "_button"` to a variable in the method and use that in the method instead.